OWML.ModLoader/Owo.cs

```diff
@@ -46,13 +46,16 @@ public void LoadMods()
                 _console.WriteLine("Verbose mode is enabled");
                 Application.logMessageReceived += OnLogMessageReceived;
             }
-            var normalMods = _modFinder.GetMods().Where(mod => !mod.Manifest.PriorityLoad).ToList();
+            var mods = _modFinder.GetMods();
+            mods.ForEach(mod => (mod as ModData).FixConfigs());
```

Comment:
if this cast bothers you I can make FixConfigs public. ModData isn't really usefull for modders either way

Reply:
yes please